tembo-cli/src/cmd/login.rs

```diff
@@ -168,3 +211,92 @@ pub fn update_access_token(
 
     Ok(())
 }
+
+pub fn update_context(org_id: &str, profile_name: &str) -> Result<(), anyhow::Error> {
```

Comment:
I think this way of writing the file would replace the entire content of the file, for example re-ordering the file, removing comments, etc. Maybe instead we rended the new part into a block string in memory, then append it to the end of the file.

Reply:
I was able to test the code and it doesn't look like it replaces nor re-orders the file but appends to the end of the file with the new profile.